### PR TITLE
Implement int8 requantization in TFLu.

### PR DESCRIPTION
**Pull request by patriklaurell**
Change-Id: I078ca4306074359d225b09a2a4c8ddb9832e9e7d

### CODE REVIEW

File: tensorflow/lite/micro/kernels/quantize.cc [hunk 1]
```diff
@@ -43,8 +43,9 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   TF_LITE_ENSURE(context, affine_quantization->scale);
   TF_LITE_ENSURE(context, affine_quantization->scale->size == 1);
 
-  TF_LITE_ENSURE(context,
-                 input->type == kTfLiteFloat32 || input->type == kTfLiteInt16);
+  TF_LITE_ENSURE(context, input->type == kTfLiteFloat32 ||
+                              input->type == kTfLiteInt16 ||
```
Comment (jenselofsson) [inline tensorflow/lite/micro/kernels/quantize.cc:47]: The indentation looks a bit off here, running clang-format will take case of that, as described in this link:
https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#c-coding-style

**Comment by gbaned** [comment]
@patriklaurell Can you please check @jenselofsson comments and keep us posted. Thanks!

File: tensorflow/lite/micro/kernels/quantize.cc (hunk 1 above)
Comment (patriklaurell) [reply tensorflow/lite/micro/kernels/quantize.cc:47]: I ran it through `clang-format --style=google` and that is the way it came out. clang-format version 6.0.0

**Comment by patriklaurell** [comment]
@jenselofsson comment has been resolved

**Comment by gbaned** [comment]
@patriklaurell Can you please resolve conflicts? Thanks!

**Pull request closed without merging**